autoload/lsp/utils.vim

```diff
@@ -24,6 +24,11 @@ function! lsp#utils#_has_textprops() abort
     return s:has_textprops
 endfunction
 
+let s:has_vim9textprops = exists('*prop_add') && has('patch-9.0.0167')
+function! lsp#utils#_has_vim9textprops() abort
```

Comment:
should we change this to `has_vimvirtualtext`. without referencing `vim9`. It will start feeling odd when we have vim10.

Reply:
might be should rename this to `lsp#utils#_has_virtual_text()` to `function! lsp#utils#_has_nvim_virtual_text()` and this to `function! lsp#utils#_has_vim_virtual_text()`?